Dockerfile

```diff
@@ -1,22 +1,44 @@
-FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base_java8:latest
-LABEL ORG="Armedia LLC" \
+FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base:latest
+
+LABEL ORG="Arkcase LLC" \
       VERSION="1.0" \
       IMAGE_SOURCE=https://github.com/ArkCase/ark_cloudconfig \
-      MAINTAINER="Armedia LLC"
+      MAINTAINER="Arkcase LLC"
+#################
+# Build JDK
+#################
+
+ARG JAVA_VERSION="1.8.0.322.b06-1.el7_9"
+
+ENV JAVA_HOME=/usr/lib/jvm/java \
+    LANG=en_US.UTF-8 \
+    LANGUAGE=en_US:en \
+    LC_ALL=en_US.UTF-8
+
+RUN yum update -y && \
+    yum -y install java-1.8.0-openjdk-devel-${JAVA_VERSION} unzip && \
+    $JAVA_HOME/bin/javac -version
+#################
+# Build ConfigServer
+#################
```

Comment:
The config server needs some configuration (like: the URL to the message broker (ActiveMQ or Kafka), and it needs the configuration files.  Does the orchestration layer (Helm chart or docker-compose file) take care of these things?